src/main/scala/com/campudus/tableaux/router/StructureRouter.scala

```diff
@@ -3,11 +3,12 @@ package com.campudus.tableaux.router
 import com.campudus.tableaux.controller.StructureController
 import com.campudus.tableaux.database.domain.{DisplayInfos, GenericTable, TableType}
 import com.campudus.tableaux.helper.JsonUtils._
-import com.campudus.tableaux.{InvalidJsonException, TableauxConfig}
+import com.campudus.tableaux.{InvalidJsonException,WrongJsonTypeException, TableauxConfig}
```

Comment:
[Minor] Missing space between `InvalidJsonException,` and  `WrongJsonTypeException`